nilearn/maskers/tests/test_surface_maps_masker.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np
+import pytest
+
+from nilearn.maskers import SurfaceMapsMasker
+from nilearn.surface import SurfaceImage
+
+
+def test_surface_maps_masker_fit_transform_shape(
+    surf_maps_img, surf_img, surf_mask
+):
+    """Test that the fit_transform method returns the expected shape."""
+    # TODO: remove after #4897 is merged
+    surf_mask = surf_mask()
+    surf_mask.data.parts["left"] = surf_mask.data.parts["left"].squeeze()
+    surf_mask.data.parts["right"] = surf_mask.data.parts["right"].squeeze()
```

Comment:
We forgot to squeeze the extra dimension in the fixtures in an older PR. So I had to do this here and in several other places.
* But #4897 is taking care of that...